vector/src/main/java/im/vector/app/core/di/ActiveSessionHolder.kt

```diff
@@ -80,18 +87,27 @@ class ActiveSessionHolder @Inject constructor(
     }
 
     fun hasActiveSession(): Boolean {
-        return activeSessionReference.get() != null
+        return activeSessionReference.get() != null || authenticationService.hasAuthenticatedSessions()
```

Comment:
an active session is now a _in memory instance_ or _cached authenticated session_

this _should_ become async with the  `hasAuthenticatedSessions` being on the IO dispatcher, however as the active session is cached in memory almost straight away in most cases I've opt'd to avoid the breaking change (will hopefully be included in the realm-kotlin changes! :crossed_fingers: )